### ISSUE TITLE
RePlaysTV doesn't record or save plays

### ISSUE THREAD

**Issue by GoldenKnight1** [lulzsun/RePlaysTV#51]
Recently, I don't know what happened, it was working well, until I run into a problem where I couldn't record manually nor automatically, plus I can't save any play I want, I tried to free some space even though I had enough, I have now 96GB and still the problem is the same, please help and always thank you for the great effort!

Also here is the log file.
[playszip-Dd9Spuq-M.zip](https://github.com/lulzsun/RePlaysTV/files/4707986/playszip-Dd9Spuq-M.zip)

**Comment by JayBoom** [comment]
Reddit thread: https://www.reddit.com/r/PlaysTV2/comments/grn9kn/question/

@lulzsun could you plz take a look at his logs?

**Comment by lulzsun** [comment]
You have an LTC process issue, which is common for some, and currently unsolvable because of the lack of knowledge on LTC and unknown root of the issue. Your best bet is to reinstall Plays and RePlays patch, making sure that you delete all files related to Plays.

These are some of the locations that you should make sure are completely deleted before starting the reinstallation:

- ~Program Files (x86)\Plays\
- %localappdata%\Plays\
- %localappdata%\Plays-ltc\

This location below is optional, since it contains your settings and video metadata files:
- %appdata%\Plays\

If a clean reinstallation does not work, then there might be a program or something affecting it, as you mentioned it occurred spontaneously. At that point, you are on your own to find out what is causing it.

**Comment by GoldenKnight1** [comment]
Thanks for the reply. So, I have to delete it and reinstall it, I've got some questions:
1- should I also delete my game replays? I've got more than 40 games, which I still didn't get the highlights from... 
2- After I uninstall, should I install the 3.0.2 version and then install the latest version?
3- If nothing changed after the re installation, will there be any hope? I really like PlaysTV the most...

**Comment by GoldenKnight1** [comment]
> I'm not lulzsun but
> 
> 1. Don't delete your games. I'm confused what u mean. Do you mean the clips? I would copy the files elsewhere and then move them back in later maybe, or just make clips manually using a video software program
> 2. Uninstall completely make sure all replays/plays files are gone, then install the playssetup.exe from the dropbox, then install the latest release of replays (which at the time I think is 3.0.3a)
> 3. Possibly with 3.0.4

Yeah, I meant the whole game, a game with more than 20 mins, i'll copy them somewhere else, and then gonna do the rest of what you just said, thank you very much!
By the way, is it true that PlaysTV is redirected to Medal.tv?

**Comment by GoldenKnight1** [comment]
> You have an LTC process issue, which is common for some, and currently unsolvable because of the lack of knowledge on LTC and unknown root of the issue. Your best bet is to reinstall Plays and RePlays patch, making sure that you delete all files related to Plays.
> 
> These are some of the locations that you should make sure are completely deleted before starting the reinstallation:
> 
> * ~Program Files (x86)\Plays\
> * %localappdata%\Plays\
> * %localappdata%\Plays-ltc\
> 
> This location below is optional, since it contains your settings and video metadata files:
> 
> * %appdata%\Plays\
> 
> If a clean reinstallation does not work, then there might be a program or something affecting it, as you mentioned it occurred spontaneously. At that point, you are on your own to find out what is causing it.

Hey there, sorry for bothering!
I did as you told me, deleted the mentioned above files and reinstalled PlaysTV, I started a League of Legends game to see if it's working and, unfortunately it didn't record, could you please check and tell me what's wrong this time?
I just want to know if it's the same problem...

[playszip-Xv5UTZwdq.zip](https://github.com/lulzsun/RePlaysTV/files/4749396/playszip-Xv5UTZwdq.zip)

**Comment by lulzsun** [comment]
The LTC process did create an unusual warning which might confirm that it is still a LTC process issue. 

`[LTCInterface.js::Proxy.onSetCaptureEngine] Invalid encoder support for capture engine 1. Using default`

I suggest trying to see if it records other games. League of Legends has always had issues with detections and recordings on some computers.

**Comment by GoldenKnight1** [comment]
Also tried Ori and the Blind Forest, Hollow Knight, still no response...
